x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -143,6 +143,7 @@ indexPattern
 
 clusterString
     : UNQUOTED_SOURCE
+    | QUOTED_STRING
     ;
```

Comment:
Since they are the same, point clusterString to indexString:
```
clusterString
  : indexString
;
```

We could fully remove it but it's worth keeping the element in for future changes.